src/test/resources/features/acceptance/pending-application.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Pending application
+  In order to apply for the Apply for Healthy Start programme
+  As a potential claimant
+  If my claim is not instantly successful I want feedback about what happens next
+
+  @RequiresWiremock
+  Scenario Outline: An application where the claimant is told we'll let you know
+    Given I am on the first page of the application
+    When I submit an application with details which need reviewing because: <reason>
```

Comment:
I wasn't particularly happy with the naming of this step so happy for any suggestions that come to mind.

Reply:
`I submit an application that doesn't get an instant answer due to: <reason>`